eng/build.ps1

```diff
@@ -0,0 +1,38 @@
+[CmdletBinding(PositionalBinding=$false)]
+Param(
+  [string] $ArchGroup,
+  [switch] $Release,
+  [switch] $c,
+  [switch] $Restore,
+  [switch] $Test,
+  [switch] $Pack,
+  [switch] $SignPackages,
+  [switch] $Publish,
+  [Parameter(ValueFromRemainingArguments=$true)][String[]]$ExtraArgs
+)
+
+$defaultargs = "-build -restore -warnaserror:0"
+
+foreach ($argument in $PSBoundParameters.Keys)
+{
+  switch($argument)
+  {
+    "Debug"        { $arguments += "-configuration Debug " }
+    "Release"      { $arguments += "-configuration Release " }
+    "ExtraArgs"    { $arguments += $ExtraArgs }
+    "c"            { $arguments += "-rebuild " }
+    "Restore"      { $arguments += "" }
```

Comment:
Why leave these empty?

Reply:
My reasoning was that Restore, Test, and Pack all happen by default, so these commands shouldn't change anything. If I just let them pass through, we'd hit https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj#L206-L218. I suppose I could allow `Restore` to pass through

Reply:
You should pass Test and Pack through as "-test" and "-pack" so when those are called only those particular targets are called. Actually for those given they are just pass through just remove them from this script and they will flow through.